examples/typescript-reactflux/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "fluxts-todomvc",
+  "version": "1.0.0",
+  "description": "Flux TodoMVC Example using Typescript",
+  "author": "Bernd Paradies",
+  "license": "MIT",
+  "repository": "https://github.com/bparadie/fluxts-todomvc",
```

Comment:
We only really need `private`, scripts and dependencies. That's how we've been doing it on all other apps as well.